spec/system/request_summary_spec.rb

```diff
@@ -3,19 +3,14 @@
 RSpec.describe "Request summary" do
   context "when logged in" do
     before do
-      visit root_path
-      click_link "Login"
-
-      fill_in "user_username", with: "bltest"
-      fill_in "user_password", with: "test"
-      click_button "Login"
+      sign_in create(:user)
     end
 
     it "displays the request summary" do
       visit account_requests_path
 
       Capybara.using_wait_time(30) do
-        expect(page).to have_css("a.active", text: "blacklight test")
+        expect(page).to have_css("a.active", text: "Test User")
```

Comment:
Would this still be Blacklight Test to match the auth_hash, or is Test User pulled from somewhere else (I'm probably just misunderstanding this so just ignore if that's the case)

Reply:
Test User is from the user factory in nla-blacklight_common.